poseidon2/src/lib.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2024 Espresso Systems (espressosys.com)
+// This file is part of the Jellyfish library.
+
+// You should have received a copy of the MIT License
+// along with the Jellyfish library. If not, see <https://mit-license.org/>.
+
+//! The Poseidon2 permutation.
```

Comment:
Could we document here a list of implemented `(field, state_size)` tuple.

Reply:
done in [6f1678a](https://github.com/EspressoSystems/jellyfish/pull/708/commits/6f1678a8afd66248c625168e07e0595472f71c6b)